charts/cloudzero-agent/README.md

```diff
@@ -111,11 +108,22 @@ helm install <RELEASE_NAME> cloudzero/cloudzero-agent \
 
 ### Metric Exporters
```

Comment:
Do we event want to confuse the customer after automatically adding this? I would think we don't need to mention ksm at all in the main readme. I would think we only need to outline how to configure the `labels` which are exported. Defaulting to `*` would get all of them. 

I would think the goal is to completely remove this section and replace it with `how to configure labels`

Reply:
I'm inclined to leave this in since we discussed still allowing users to use there own KSM, per this AC point:

> Documentation is added that shows more technical users how they can use their own KSM

If we want to say "no custom KSM" at all, then I think removing this section makes sense.

Reply:
One of the primary challenges we have with improving user experience is how to make the installation guide more consumable. This has been made more challenging with "bring your own KSM," which is the point of us taking it into our chart. 

If we feel strongly, I would recommend adding any special configuration to support `bring your own KMS` to a guide under docs, and minimize the configuration in the primary readme.